kubeflow/trainer/options/__init__.py

```
@@ -24,29 +24,37 @@
-    "PodTemplateOverride",
-    "PodTemplateOverrides",
-    "SpecAnnotations",
-    "SpecLabels",
+    "PodSpecPatch",
+    "PodTemplatePatch",
+    "ReplicatedJobPatch",
+    "RuntimePatch",
     "TrainerArgs",
     "TrainerCommand",
+    "TrainingRuntimeSpecPatch",
 ]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)